### ISSUE TITLE
Animation of top bars on hide/show keyboard

### ISSUE THREAD

**Issue by mdznr** [peterhajas/Genesis#11]
The animation of the top bars on hiding and showing the keyboard is off. There is a period of time where white it shown behind them. They should stick together as they animate.

**Comment by peterhajas** [comment]
Could you be more clear? Do you mean the status bar and the navigation controller? What is shown behind them?

If it helps, tapping the shift key 5 or so times in the Simulator will slow down animations.

**Comment by mdznr** [comment]
Correct, the navigation controller and the status bar. When showing the keyboard, the navigation controller slides under the status bar before the status bar moves off screen. When hiding the keyboard, the navigation controller pops out jumps out, leaving an empty white area where the status bar will eventually animate to.
